package.json

```diff
@@ -84,6 +84,7 @@
     "@types/react-table": "7.0.24",
     "@types/stream-buffers": "3.0.3",
     "@xstate/react": "0.8.1",
+    "@types/crypto-js": "4.0.1",
```

Comment:
Should be a dev dep.

Reply:
Going to solve this in another project/task by minimising the final docker build image.